src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2800,7 +2800,7 @@ class misc_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDamageCommand(ChatHandler* handler, uint32 damage)
+    static bool HandleDamageCommand(ChatHandler* handler, uint32 damage, Optional<std::string> percent)
```

Comment:
Does this need an update elsewhere, too, to forward the value of "percent" to this function? Or will it just work out of the box

Reply:
It will work out of the box